compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -94,6 +94,8 @@ OMR::ARM64::CodeGenerator::initialize()
 
    cg->setSupportsRecompilation();
 
+   cg->setSupportsPrimitiveArrayCopy();
```

Comment:
Shouldn't this be set in a downstream project that provides support for it?  At the moment, this PR can set up a call to the TR_ARM64arraycopy assembler helper, but OMR does not build this helper in meaning that call will always crash.  It is probably best to leave enabling it to a downstream project that does build in the helper.

Granted, other architectures set this up in OMR, but I'm not sure that is the right thing to do.

Reply:
OK, I will move it to the downstream project.

Reply:
Moving `setSupportsPrimitiveArrayCopy()` to the downstream project does not work.
I haven't identified the reason yet, but arraycopy IL nodes are not generated in that case.

Reply:
I moved `cg->setSupportsRecompilation();` to https://github.com/eclipse-openj9/openj9/pull/13263